jdaviz/configs/default/plugins/plot_options/plot_options.vue

```diff
@@ -783,6 +792,23 @@ module.exports = {
     contour_custom_levels_set_value(e) {
       this.contour_custom_levels_txt = e
       this.contour_custom_levels_value = e.split(',').filter(n => n.trim().length).map(n => Number(n)).filter(n => !isNaN(n))
+    },
+    colorStyle(item, cmap_samples) {
+      var cmap_strip_width = 3

```

Comment:
Could probably also generalize this to compute from the length of `cmap_samples` instead

```suggestion
      var cmap_strip_width = 2  # 2 percent for 50 color samples
```